### ISSUE TITLE
Enums in schema is not generated probably

### ISSUE THREAD

**Issue by mohammed-alfarra** [ahmedesa/laravel-api-tool-kit#31]
I tried to use this command to create new Product model with migration file but Enum type is not generated probably.

**Command:**
```bash
php artisan api:generate Product "name:string:unique|description:text:nullable|image:string|status:enum:in-stock,finished" -mf
```

**Result:**
```php
public function up(): void
{
    Schema::create('products', function (Blueprint $table) {
        $table->id();
	$table->string('name')->unique();
	$table->text('description')->nullable();
	$table->string('image');
        // Attention: Check the syntax for enum definition below
	$table->enum('status')->in-stock,finished();
            
        $table->timestamps();
     });
}
```

**Comment by ahmedesa** [comment]
Thanks for pointing out the migration issue. Currently, Laravel's schema builder doesn't support the enum type directly. We're aware of this limitation and are actively working on finding a solution. In the meantime, we can use the string type and enforce enum values through validation. Let me know if you need further help or updates on this

**Comment by ahmedesa** [comment]
the issue has been fixed with the tag [2.1.1](https://github.com/ahmedesa/laravel-api-tool-kit/releases/tag/2.1.1)

Here is how to use it

```
php artisan api:generate Product "status:enum(in-stock,finished)"
```
